projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1441,5 +1453,6 @@ protected static function build_sprintf( $args ) {
  * @return string
  */
 function publicize_calypso_url() {
+	_deprecated_function( __METHOD__, 'jetpack-10.9', 'Publicize_Base::publicize_calypso_url' );
```

Comment:
You probably want to change `Publicize_Base` to `Publicize` here, since that's the class to actually call the function from.